notebooks/EOPF_Tutorials/EOPF_SAFE_ZARR_COMPARE_METADATA_SENTINEL2_L2A.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    for attribute in root_group.GetAttributes():
</pre><p>Here you work with  <code>root_group</code>  but in the previous cell you define <code>rootgroup</code> , which means that the current code is not reading the SAFE metadata but the Zarr twice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EOPF-Sample-Service/eopf-sample-notebooks/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>